src/errors.ts

```diff
@@ -230,3 +230,15 @@ export class CommentUpvoteNotFound implements DropboxReadsError {
     this.type = this.constructor.name;
   }
 }
+
+export class ObjectAlreadyExists implements DropboxReadsError {
+  readonly message?: string = "Comment upvote not found";
```

Comment:
Something along lines of "Object already exists"

Reply:
oop, i suppose this is what happens when ending the day before finishing to modify a copy paste 😅